client/src/components/ChatItem.tsx

```diff
@@ -1,35 +1,114 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { auth } from '../firebase/connection';
+import { useAuthState } from 'react-firebase-hooks/auth';
+import OrBackground from '../assets/OrBackground.svg';
 import { createSearchParams, useNavigate } from 'react-router-dom';
+import { getRealtimeLastMessage, getUnreadMessagesCount } from '../helpers/chatFunctions';
+import { Message } from '../firebase/message';
+import { getCurrDateIsrael, isYesterday } from '../helpers/dateFunctions';
 
 export interface ChatItemProps {
-  name: string;
-  isEnded: boolean;
-  chatId: string;
-}
+  name: string,
```

Comment:
I am avoiding calling it something more specific in case we'd at some point want to use that component for the other role as well,
and I actually think that from the small "component's point of view" it's pretty clear :)